src/languages/bash.js

```diff
@@ -38,6 +38,18 @@ export default function(hljs) {
     end: /\)/,
     contains: [ hljs.BACKSLASH_ESCAPE ]
   };
+  const COMMENT = hljs.inherit(
+    hljs.COMMENT(undefined, undefined),
+    {
+      match: [
+        /(^|\s)(?=#)/,
```

Comment:
```suggestion
        /(^|\s)/,
```

Multi-match handles this, you don't need the look-ahead anymore.

Reply:
Ahh, awesome. Fixed!